src/main/resources/site/pages/default/default.es6

```diff
@@ -226,7 +226,7 @@ exports.get = function(req) {
     .setId('breadcrumbs')
     .uniqueId()
 
-  const hideBreadcrumb = page.page.config.hide_breadcrumb ? page.page.config.hide_breadcrumb : false
+  const hideBreadcrumb = page.page.config.hide_breadcrumb && page.page.config.hide_breadcrumb
```

Comment:
should we use `!!page.page.config.hide_breadcrumb` instead? using && on the same value twice is semantically weird.